fontdrasil/build.rs

```diff
@@ -0,0 +1,82 @@
+//! Generate static arrays with data from the Adobe Glyph Lists
+
+use std::{env, fs::File, io::BufWriter, path::Path, str::FromStr};
+
+const OUT_FILE: &str = "agl_codegen.rs";
+
+fn main() {
+    println!("cargo::rerun-if-changed=data");
```

Comment:
Not data/ ?

Reply:
seems fine: https://doc.rust-lang.org/1.77.2/cargo/reference/build-scripts.html#rerun-if-changed